src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/_params.py

```diff
@@ -0,0 +1,31 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+#pylint: disable=line-too-long
+
+from azure.cli.commands.parameters import (location_type,)
+from azure.cli.commands import register_cli_argument, CliArgumentType
+
+
+name_type = CliArgumentType(
+    options_list=('--name', '-n'),
+    help='Name for new Azure IoT Hub.')
+sku_type = CliArgumentType(
+    choices=['F1', 'S1', 'S2', 'S3'], default='F1',
```

Comment:
>['F1', 'S1', 'S2', 'S3'], [](start = 12, length = 25)

In general, we try to reflect on the values in the SDK, the reason being that if you added a SKU to your swagger and thus the SDK, it would automatically be reflected in the CLI. Like this, you would have to remember to add the new value to this choice list in order to enable it in the CLI. You should be able to import IotHubSku from ios_hub_client_enums. The azure.cli.commands.parameters package has the 'get_enum_choices' helper that you can use. So your  definition would look like:

sku_type = CliArgumentType(choices=get_enum_choices(IotHubSku), help='stuff')